cvc4/src/cvc4_solver.cpp

```diff
@@ -827,6 +856,12 @@ Term CVC4Solver::make_term(Op op,
   }
   catch (::CVC4::api::CVC4ApiException & e)
   {
+    if (op.prim_op == Forall || op.prim_op == Exists)
+    {
+      throw IncorrectUsageException(
```

Comment:
Why is that? The standard allows multiple.

Reply:
Because it got really complicated to support term traversal if I allowed more than one. CVC4 uses a `BOUND_VAR_LIST` that would have to be flattened. Or if we had a smt-switch version of `BOUND_VAR_LIST` then we'd have to transform all the other solvers to this representation. I'm open to other ideas, but this was the simplest thing I could think of. Then I just always assume that the term is `quantifier param . term`. And it's still equally expressive, because you can always nest them.